### DISCUSSION TITLE
the fence manager does not start

### DISCUSSION THREAD

**Discussion by brano1990** [swar/Swar-Chia-Plot-Manager#348] [Q&A]
hi it doesn't work for me
python manager.py start

i can't find a directory in linux Ubuntu
I have no idea where to look for my directory

# LINUX EXAMPLE: /usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon/chia
# LINUX2 EXAMPLE: / home / swar / chia-blockchain / venv / bin / chia

 probably this is my directory  chia_location:/  home/pc/chia-blockchain/chia
FileNotFoundError: [Errno 2] No such file or directory: '/home/pc/chia-blockchain/app.asar.unpacked/daemon

`(venv) pc@pc-:~/Stiahnuté/Swar-Chia-Plot-Manager-main$ python manager.py start
Traceback (most recent call last):
  File "manager.py", line 29, in <module>
    start_manager()
  File "/home/pc/Stiahnuté/Swar-Chia-Plot-Manager-main/plotmanager/library/utilities/commands.py", line 47, in start_manager
    raise ManagerError('Failed to start Manager. Please look at manager.log for more details on the error. It is in the same folder as manager.py.')
plotmanager.library.utilities.exceptions.ManagerError: Failed to start Manager. Please look at manager.log for more details on the error. It is in the same folder as manager.py.
(venv) pc@pc~/Stiahnuté/Swar-Chia-Plot-Manager-main$ 
`

**Comment by swar** [comment]
Please use Github Discussions next time. This is not an Issue with the software.

**Comment by swar** [comment]
You need to confirm your file actually exists.

**Comment by brano1990** [comment]
Where should I use the discussion? I can't find a director in Linux. I would need a real example Windows works but linux doesn't
